fred2/missionsave.cpp

```diff
@@ -2435,15 +2438,34 @@ int CFred_mission_save::save_mission_info()
 			fout(")");
 		}
 
-		// squadron wings
-		if (strcmp(Squadron_wing_names[0], "Alpha") || strcmp(Squadron_wing_names[1], "Beta") || strcmp(Squadron_wing_names[2], "Gamma") || strcmp(Squadron_wing_names[3], "Delta") || strcmp(Squadron_wing_names[4], "Epsilon")) {
+		// We are now always writing multi squadron wings to avoid a multi bug where, if the mission designer skips one of the squadron wings
+		// the wing after would disappear from the mission
+		if (The_mission.game_type & MISSION_TYPE_MULTI) {
```

Comment:
Add a condition here to check that the mission save format is not retail.  If someone ever wanted to make a retail-compatible multi mission, the "output only if they are not the default" behavior should be used, otherwise the retail engine would choke on the unexpected data.